daktari/checks/google.py

```diff
@@ -59,30 +59,28 @@ class DockerGoogleCloudAuthConfigured(Check):
     name = "google.dockerGCloudAuthConfigured"
     depends_on = [GoogleCloudSdkInstalled]
 
-    def __init__(self, cloud_project, region, region_number):
+    def __init__(self, cloud_project, region, registry="europe-west2-docker.pkg.dev"):
```

Comment:
Shall we const the registry? Seems to be repeated in a few places.

Reply:
Actually thinking about this, shouldn't we remove the default value as this is meant to be reusable by people Glean / not glean. So we should always be passing in a registry in our .daktari.py files.